x-pack/plugins/session_view/common/utils/flatten_json.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const flattenJSON = (data: any) => {
+  const result: any = {};
+  function recurse(cur: any, prop: string) {
+    if (Object(cur) !== cur) {
```

Comment:
what js magic are we doing here? can we type check using `typeof` instead?

Reply:
I think its trying to check whether or not the current iterated property in the JSON is a nested object but not an array, so `typeof` wouldn't work

**But** this function is not needed anymore, i will remove this file.